internal/server/models/deviceflowauthrequest.go

```diff
@@ -11,16 +11,8 @@ type DeviceFlowAuthRequest struct {
 	UserCode   string
 	DeviceCode string
 
-	AccessKeyID uid.ID
-
-	// AccessKeyToken is set once the request is approved.
-	AccessKeyToken EncryptedAtRest
-
 	ExpiresAt time.Time
 
-	// AccessKey will be populated by some queries, but is never used on writes.
-	AccessKey *AccessKey `db:"-"`
-
-	// Organization will be populated by some queries, but is never used on writes.
-	Organization *Organization `db:"-"`
+	// ApprovedWith is the access key that approved this request
```

Comment:
```suggestion
	// ApprovedWith is the ID of the access key that approved this request
```

This looks alarming otherwise.